nemo/collections/nlp/models/ner_model.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict, Optional
+
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+# TODO replace with nemo module
+# from transformers import BertModel
+from nemo.collections.nlp.modules.common.huggingface import BertEncoder
+
+from nemo.collections.common.tokenizers.bert_tokenizer import NemoBertTokenizer
+from nemo.collections.nlp.data.token_classification_dataset import BertTokenClassificationDataset
+
+__all__ = ['NERModel']
+
+from nemo.collections.nlp.modules.common import TokenClassifier
+
+
+
+
+class NERModel(pl.LightningModule):
```

Comment:
should inherit from nemo.core.classes.modelPT.ModelPT
Also, decorate with @experimental